updater/reports/ReportFailedWebhooks.py

```diff
@@ -0,0 +1,14 @@
+from .ReportDaily import *
+
+# Report how many webhooks failed
+class ReportFailedWebhooks(ReportDaily):
+	def name(self):
+		return "failed-webhooks"
+
+	def updateDailyData(self):
+		self.detailedHeader, self.detailedData = self.parseData(
+			self.executeScript(self.scriptPath("failed-webhooks.sh")))
+		self.header = ["date", "failed webhooks"]
+		self.data.append([str(self.yesterday()), len(self.detailedData)])
```

Comment:
```suggestion
		self.data.append(
			[
				str(self.yesterday()),
				sum(map(lambda x: int(x[4] if len(x) > 3 else 0), newData)),
			])
		self.detailedData = newData[:25]
```
After looking at this with fresh eyes, I think here is a little bug.

For the chart we want to count the number of failures *for each* failed webhook (IIRC the count column is `4`). That's why we need the sum function.

For the detailed view (the table). We just want to write out the results. Since on a large instance many many hooks could fail we limit the length of the list to 25 here.

Disclaimler: This is untested and just "code in the GitHub review window" 😨